docs/book/v3/features/template/plates.md

```diff
@@ -47,3 +47,86 @@ $plates->loadExtension(new CustomExtension());
 // Inject:
 $renderer = new PlatesRenderer($plates);
 ```
+
+## Configuration and Factory
+
+zend-expressive-platesrenderer ships with the factory
+`Zend\Expressive\Plates\PlatesRendererFactory`, which will both create the
+Plates engine instance, and the `PlatesRenderer` instance. If you are using
+[zend-component-installer](https://docs.zendframework.com/zend-component-installer/),
+this will be automatically wired for you during installation.
+
+The factory looks for the following configuration in the `config` service, using
+any it finds:
```

Comment:
I think a filename as example or the default filename should be added here. Like "(e.g. `config/plates.config.php`)"